src/scss/layout/_layout.scss

```diff
@@ -57,35 +43,73 @@ article {
 			display: table;
 		}
 	}
-}
 
-/*
- * Default WP Alignment Classes
+	/*
```

Comment:
It would be nice to use comment styles that have been applied elsewhere here, for example: `/** Customized styles for WP content image selectors **/`.